addons/udes_stock_refactoring/models/stock_move.py

```diff
@@ -151,13 +151,11 @@ def _action_confirm(self, *args, **kwargs):
     def _action_assign(self):
         """
         Extend _action_assign to trigger refactor action.
-        n.b. _action_assign does not return anything in core Odoo, so we
-        don't return any extra moves that may have been created
-        by refactoring.
         """
         res = super(StockMove, self)._action_assign()
 
-        self._action_refactor(stage="assign")
+        refactored_moves = self._action_refactor(stage="assign")
```

Comment:
Shouldn't this be `refactored_moves = res._action_refactor(stage="assign")` ?

Reply:
Fixed in https://github.com/unipartdigital/udes-open/pull/745/commits/ff1084d3a25aaa287caf20b0ef2590fbff124df4